index.html

```diff
@@ -69,7 +69,7 @@
       <script async="true" id="_carbonads_js" src="//cdn.carbonads.com/carbon.js?zoneid=1673&serve=C6AILKT&placement=adamschwartzco"></script>
     </div>
 
-    <script src="js/deps/draggabilly.js"></script>
+    <script src="https://unpkg.com/draggabilly@2/dist/draggabilly.pkgd.min.js"></script>
```

Comment:
And this locked to `https://unpkg.com/draggabilly@2.1.1/dist/draggabilly.pkgd.min.js`.